src/sqlfluff/dialects/dialect_redshift.py

```diff
@@ -16,15 +16,14 @@
     Delimited,
     Nothing,
     OptionallyBracketed,
+    Matchable,
 )
-
 from sqlfluff.core.dialects import load_raw_dialect
-
 from sqlfluff.dialects.dialect_redshift_keywords import (
     redshift_reserved_keywords,
     redshift_unreserved_keywords,
 )
-
+from sqlfluff.dialects.dialect_ansi import ObjectReferenceSegment
```

Comment:
Is this needed? Surely it already exists in this copy of ANSI?

Reply:
Good point. Since the Redshift dialect inherits for Postgres (which inherits from ANSI), we can allow `ColumnReferenceSegment` in the Redshift dialect to inherit `ObjectReferenceSegment`s in the dialect by using the
`redshift_dialect.get_segment("ObjectReferenceSegment")` method. This is a much more maintainable pattern than directly importing the segment class from the `dialect_ansi.py` module.

Change made.